torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2009,6 +2009,8 @@ def __init__(
         # Maps symbolic ints to their original concrete values
         # Currently populated from tensors
         self.var_to_val: Dict[sympy.Symbol, sympy.Integer] = {}
+        # Maps symbolic ints to their size hints. Only used for unbacked symints.
```

Comment:
If it's only used for unbacked symint, the name should probably reflect it. Also, you seem to be accessing this field directly from outside of symbolic_shapes; mediate it through a method or something please.

Reply:
(I'm expecting some mechanism to populate this by user annotation eventually. Also, it would be good to explain in the comment here what this is being used for.)